javascript/builtins/Intl/NumberFormat.json

```diff
@@ -510,7 +510,7 @@
                 "description": "<code>options.trailingZeroDisplay</code> parameter",
                 "support": {
                   "chrome": {
-                    "version_added": false
+                    "version_added": "106"
```

Comment:
Did you test this?  If so, could you provide your testing code so we may verify?

Reply:
Sure, [here's a page for that](https://cattegat.github.io/test-trailingzerodisplay/).

```js
Intl.NumberFormat(navigator.language, {
    style: "currency",
    currency: "SEK",
    currencyDisplay: "symbol",
    trailingZeroDisplay: "stripIfInteger"
}).format(123.00);
```